.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: build and push docker image
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.0.3
+      - uses: actions/cache@v1
+        with:
+          path: ~/operator-sdk/bin/
+          key: operator-sdk-${{ env.OPERATOR_SDK_VERSION }}
+      - name: setup operator-sdk
+        run: |
+          if [ ! -x ~/operator-sdk/bin/operator-sdk ]; then
+            mkdir -p ~/operator-sdk/bin/
+            curl -L https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu > ~/operator-sdk/bin/operator-sdk
+            chmod +x ~/operator-sdk/bin/operator-sdk
+          fi
+          echo "::add-path::${HOME}/operator-sdk/bin/"
+      - uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-build-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-build-
+      - name: Build with operator-sdk
+        run: operator-sdk build quay.io/piraeusdatastore/piraeus-operator:latest
+      - name: Push docker image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
Do these secrets already exist?

Reply:
No :sweat_smile:
Automatic push is the only thing I could not really test in my fork (docker build works fine)

Reply:
@rck Could you configure secrets for pushing to `quay.io`? I don't have the permissions.

Reply:
I will take care of that.

Reply:
The secrets now exist